assets/gui/start-screen.json

```diff
@@ -3,16 +3,34 @@
         {
             "name": "start-button",
             "path": "assets/gui/start-button.png"
+        },
+        {
+            "name": "start-selected-button",
+            "path": "assets/gui/start-selected-button.png"
+        },
+        {
+            "name": "load-button",
+            "path": "assets/gui/load-button.png"
         }
     ],
     "buttons": [
         {
-            "selected_image_asset": "start-button",
+            "selected_image_asset": "start-selected-button",
             "unselected_image_asset": "start-button",
-            "name": "start-button",
+            "name": "start-selected-button",
```

Comment:
Let's keep the name the same. This is a "start button", not a "start selected button".

The asset name can change, which is exactly what we want.